Winium/Winium.sln

```diff
@@ -55,58 +57,169 @@ Global
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
 		Debug|Any CPU = Debug|Any CPU
+		Debug|ARM = Debug|ARM
```

Comment:
When building Debug or Release configuration it will build for all platforms (x86, arm, anycpu), which is redundant in our case. You will also have to maintain these configuration when adding new projects, etc, because sometime VS does not add new projects to each configuration correctly.